guide/commando/README.md

```diff
@@ -0,0 +1,142 @@
+
+The final option, `invite`, is the **full** invite URL to your bot's support server. While not a required option, it's a good idea to have a support server on hand to handle questions and concerns your users may have about your bot if it is public. If not, it's safe to leave this option out.
+
+Next you're going to register the command groups, args types, and default commands. Then, you register the commands in a folder. You can name the folder whatever you want, but I personally recommend `commands`, as it makes the most sense.
+
+```js
+client.registry
+	.registerDefaultTypes()
+	.registerGroups([
+		['first', 'Your First Command Group'],
+		['second', 'Your Second Command Group'],
+	])
+	.registerDefaultGroups()
+	.registerDefaultCommands()
+	.registerCommandsIn(path.join(__dirname, 'commands'));
+```
+
+Remember how I said you'd need path? There's the answer to your burning question.
+
+Doing this you've also created your first command group! Make another folder called `first` in your `commands` folder, so you can put the commands for that group in there. The group will be displayed in the help command as `Your First Command Group`. You can use any name you want for either of these options, but do note that the key (`first`) **must be lowercase**!
+
+Adding more command groups is as simple as adding another option to the array and making another folder.
+
+<!-- eslint-skip -->
+```js
+.registerGroups([
+	['first', 'Your First Command Group'],
+	['second', 'Your Second Command Group'],
+	['third', 'Your Third Command Group'],
+])
+```
+
+Should you want to disable a default command, such as if you wanted to make your own help command and replace the default one, you can pass that as an option in `registerDefaultCommands`.
+
+<!-- eslint-skip -->
+```js
+.registerDefaultCommands({
+	help: false,
+})
+```
+
+Next, you're going to need to create a ready event and an error event, as usual.
+
+```js
+client.on('ready', () => {
+	console.log(`Logged in as ${client.user.tag}! (${client.user.id})`);
+	client.user.setActivity('with Commando');
+});
+
+client.on('error', console.error);
+```
+
+This will send a message to your console when the bot is ready, and set the Bot's playing status to `with Commando`. You can set both to whatever you wish.
+
+Last but certainly not least, let's log the bot in.
+
+```js
+client.login('Your Secret Token');
+```
+
+<p class="danger">You should use environment variables or a `config.json` for your token instead of passing it directly!</p>
+
+And there you have it! You've set up your `index.js` file! In the end your file structure should look like this, along with whatever `.gitignore` or `config.json` you may have:
+
+```
+commands
+- first
+index.js
+package.json
+```
+
+Your `index.js` file should look something like this:
```

Comment:
This and any other "final" snippets at the end of pages should go in the `/code-samples` folder, preferably in a `/commando` subfolder. Be sure to copy the "Resulting code" header and text from the other pages.